docs/psf_grids.rst

```diff
@@ -0,0 +1,59 @@
+*****************
+Using PSF Grids
+*****************
+
+WebbPSF includes functionality designed to work with the Photutils package to
+enable precise PSF-fitting photometry and astrometry. This makes use of the
+`GriddedPSFModel` class (available in Photutils > 0.6), which implements a
+version of the empirical or effective PSF ("ePSF") modeling framework pioneered by Jay
+Anderson, Ivan King, and collaborators. This approach has been highly successful with
+HST and other space observatories, and we expect it will also be productive with JWST. 
+In practice we will want to use ePSF models derived from real observations, but for now
+we can make them in simulation. 
+
+The first step is to create a grid of fiducial PSFs spanning the
+instrument/detector of choice. This can be done using the :meth:`~webbpsf.JWInstrument.psf_grid` method
```

Comment:
I previewed this document and I think something is off with the formatting in the :meth:`~webbpsf.JWInstrument.psf_grid` line.

Reply:
I can't figure this one out. The syntax is right, but Sphinx is not hyperlinking correctly. I suspect this has something to do with how psf_grid is inherited from a superclass in poppy, so it isn't actually in webbpsf to link to? Probably needs some advanced trickery with intersphinx to work correctly, which I'm not going to prioritize right now.